### DISCUSSION TITLE
DetailedCellError with Built-In SEARCH Function

### DISCUSSION THREAD

**Discussion by haraleib** [handsontable/hyperformula#689] [General]
Hello,

I have a problem using the Hyperformula built-in `SEARCH` function, if the first parameter is not within the second parameter.
As you see in the screenshot, if the first parameter is not in the second parameter it results in a `DetailedCellError`. My workaround is to wrap the statement in a `IFERROR` function.
Is the `DetailedCellError` intentional ? Do you provide a common way to handle `DetailedCellErrors `properly when evaluating expressions? 
![image](https://user-images.githubusercontent.com/19574757/120978041-57e9b580-c774-11eb-8e4f-910b2c689947.png)

**Comment by izulin** [comment]
This behaviour (`SEARCH` returning `error`) is according to specification, see for reference: https://support.microsoft.com/en-us/office/search-searchb-functions-9ab04538-0e55-4719-a72e-b6f54513b495

**Reply by haraleib** [reply]
Ok, thank you for your reply.
